CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## v0.0.2 [WIP]
+
+_What's new?_
+
+- New tables added
+  - [snowflake_account_parameter](https://hub.steampipe.io/plugins/turbot/snowflake/tables/snowflake_account_parameter)
+  - [snowflake_login_history](https://hub.steampipe.io/plugins/turbot/snowflake/tables/snowflake_login_history)
+  - [snowflake_schemata](https://hub.steampipe.io/plugins/turbot/snowflake/tables/snowflake_schemata)
+  - [snowflake_session](https://hub.steampipe.io/plugins/turbot/snowflake/tables/snowflake_session)
+
+_Enhancements_
+
+- Added `account` and `region` metadata column to all existing tables.
+- Changed the column type of `ext_authn_duo` column from string to bool in `snowflake_user` table.
```

Comment:
I think this should be under the `_Bug fixes_` section instead, assuming it was always meant to be a bool instead of string type